benchmark/run_jane4.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run JANE 4 software
+gene_tree_dir=$1
+output_fp=$2
+verbose=$3
+stdout=$4
+stderr=$5
+scripts_dir=$6
+species_tree_fp=$7
+input_file_nex=$8
+output_file=$9
+jane_install_dir=${10}
+
+total_user_time_jane="0.0"
+total_wall_time_jane="0.0"
+i=0
+printf "#JANE4\n" >> $output_fp
```

Comment:
This file gets removed at the end, so it can be temporal, right? So no need to pass it. Also it could be a text variable, right?